lib/cadet_web/router.ex

```diff
@@ -33,6 +33,13 @@ defmodule CadetWeb.Router do
 
     resources("/sourcecast", SourcecastController, only: [:create, :delete])
 
+    get("/achievements/", AchievementsController, :index)
+    post("/achievements/", AchievementsController, :update)
+    post("/achievements/update/", AchievementsController, :edit)
+    delete("/achievements/", AchievementsController, :delete)
+
+    delete("/achievements/goals/", AchievementsController, :delete_goal)
```

Comment:
Instead of specifying the achievement ID in the payload, they should be in the URL:

```
POST /achievements/:id
DELETE /achievements/:id
DELETE /achievements/:id/goals
```